x-pack/plugins/ml/server/models/data_recognizer/modules/security_windows/manifest.json

```diff
@@ -30,84 +30,119 @@
             ]
           }
         }
-      ],
-      "must_not": { "terms": { "_tier": [ "data_frozen", "data_cold" ] } }
+      ]
     }
   },
   "jobs": [
     {
-      "id": "v2_rare_process_by_host_windows_ecs",
-      "file": "v2_rare_process_by_host_windows_ecs.json"
+      "id": "v3_windows_anomalous_service",
+      "file": "v3_windows_anomalous_service.json"
     },
     {
-      "id": "v2_windows_anomalous_network_activity_ecs",
-      "file": "v2_windows_anomalous_network_activity_ecs.json"
+      "id": "v3_windows_rare_user_runas_event",
+      "file": "v3_windows_rare_user_runas_event.json"
     },
     {
-      "id": "v2_windows_anomalous_path_activity_ecs",
-      "file": "v2_windows_anomalous_path_activity_ecs.json"
+      "id": "v3_windows_rare_user_type10_remote_login",
+      "file": "v3_windows_rare_user_type10_remote_login.json"
     },
     {
-      "id": "v2_windows_anomalous_process_all_hosts_ecs",
-      "file": "v2_windows_anomalous_process_all_hosts_ecs.json"
+      "id": "v3_rare_process_by_host_windows_ecs",
```

Comment:
As with the `security_linux` jobs, I'd recommend removing the `_ecs` suffix from all job and datafeed IDs and filenames. Unless there is a specific reason why you'd want to retain it for only certain jobs in the module?